todo_react/todo-react/src/App.tsx

```diff
@@ -0,0 +1,30 @@
+import React, {useEffect, useState} from 'react';
+import './App.css';
+import {AddToDo} from "./components/AddToDo";
+import {ToDoItem} from "./components/ToDoItem";
+import {useTodoContext} from "./context/context";
+
+export const App: React.FC = () => {
+    const todosContext = useTodoContext()
+    useEffect(() => {
```

Comment:
`<App>` is consumer of `todos` logic. Isn't better to manage all related logic (`useEffect` in this case) in `<TodoContextProvider>`?